pkg/clients/aws.go

```diff
@@ -55,9 +55,28 @@ import (
 // DefaultSection for INI files.
 const DefaultSection = ini.DefaultSection
 
-// GlobalRegion is the region name used for AWS services that do not have a notion
-// of region.
-const GlobalRegion = "aws-global"
+// GetGlobalRegion returns the global region for AWS services that do not have a notion
+// of region.  Ex. iam
+// Defaults to 'aws' partition
+func GetGlobalRegion(partition string) string {
+	// Global regions are convention based by partition
+	// Ex. "aws-global"
+	if partition == "" {
+		return "aws-global"
+	}
+	return partition + "-global"
```

Comment:
I'd prefer us not making assumptions here and let them know the format in the documentation. There might be cases where they use a string that shouldn't have `-global` appended to it.